scripts/coverage_report_jenkins.py

```diff
@@ -27,12 +27,10 @@
 # log to stdout too
 add_stdout_handler(logger)
 
-# Global variables
+# Globals
+# TODO:  Make these config items
 coverage_dir = '/coverage'
-scan_timeout = env.sonarqube.get('scan_timeout', 600)
```

Comment:
Question: Why we need to move those?

Reply:
They didn't need to be global.

Reply:
But since I was adding yet more config items, and didn't want them to be global, I made all the config items consistently not global.